cmd/goacme/main.go

```diff
@@ -130,6 +131,15 @@ type command struct {
 	// 'goacme help <command>' output.
 	// The template context is longTemplateData.
 	Long string
+
+	// ConfigDir is the configuration directory which is used
+	ConfigDir string
+
+	// DefaultDisco is the default CA directory endpoint.
+	DefaultDisco string
+
+	// AccountFile is the default user config file name.
+	AccountFile string
```

Comment:
There's no other way to pass it to the template for printing that I know of. If you have another way to do it then we can go that route, but honestly with the way this was architected this is the most concise change to fix the issue that I am aware. Honestly, the only two I hesitated about though were the default disco and account file. I don't agree it's substantial to add ConfigDir at all though.